src/Commands/Storage/Blob/Container/ContainerDetailsCommand.cs

```diff
@@ -23,9 +23,9 @@ public sealed class ContainerDetailsCommand(ILogger<ContainerDetailsCommand> log
         Requires {OptionDefinitions.Storage.AccountName} and {OptionDefinitions.Storage.ContainerName}.
         """;
 
-    public override string Title => _commandTitle;
+    public override string Title => CommandTitle;
 
-    [McpServerTool(Destructive = false, ReadOnly = true)]
+    [McpServerTool(Destructive = false, ReadOnly = true, Title = CommandTitle)]
```

Comment:
Let's add a test that ensures that all ExecuteAsync commands decorated with the `McPServerTool` attribute include a Title property.

Reply:
Added